client/reader/search-stream/style.scss

```diff
@@ -34,6 +34,11 @@
 	width: 830px;
 }
 
+/* clamp to 3 lines for search-rec excerpts */
+ .is-reader-page .search-stream .reader-related-card-v2__excerpt.post-excerpt {
+	max-height: 15px * 1.6 * 3;
```

Comment:
Looks like it's being applied, but perhaps starts too late for these cards? It often seems to kick in after the `...`:

<img width="853" alt="screen shot 2016-12-12 at 14 36 35" src="https://cloud.githubusercontent.com/assets/17325/21085573/a245f466-c078-11e6-901a-d9ff454cb0d3.png">